config/config.go

```diff
@@ -577,6 +580,9 @@ func (cfg *MempoolConfig) ValidateBasic() error {
 	if cfg.CacheSize < 0 {
 		return errors.New("cache_size can't be negative")
 	}
+	if cfg.MaxBytes <= 0 {
+		return errors.New("max_bytes must be a positive number")
```

Comment:
Shouldn't minimum mempool size be enough to store a single transaction? Maybe minimum mempool size should be `maxTxSize`?

Reply:
Don't think we should handle such cases (clearly misconfiguration)